packages/now-cli/src/util/dev/server.ts

```diff
@@ -744,9 +743,9 @@ export default class DevServer {
       this.getLocalEnv('.env', nowConfig.env),
       this.getLocalEnv('.env.build', nowConfigBuild.env),
     ]);
-    Object.assign(process.env, buildEnv);
```

Comment:
@TooTallNate @lucleray Do you know why we were assigning to `process.env` here? Line 1691 is also passing `process.env` so I'm not sure if this is still necessary.

Reply:
Not really sure… perhaps it was from when we were running the builders in the same process as the now dev server.

Reply:
Hmm, that makes sense. I'll leave this one just in case because I don't think it will cause any harm.